core/server/web/api/v2/admin/routes.js

```diff
@@ -149,6 +149,13 @@ module.exports = function apiRoutes() {
         mw.authenticateClient('Ghost Backup'),
         apiv2.http(apiv2.db.backupContent)
     );
+    router.get('/db/import', mw.authAdminApi, apiv2.http(apiv2.db.asyncImportStatus));
```

Comment:
Since this is APIv2, it might be possible to alter the POST .../db method once changes are made to the admin client